tasks/changelog.rb

```diff
@@ -30,7 +30,10 @@ def write
     end
 
     def path
-      format(ENTRIES_PATH_TEMPLATE, type: type, name: str_to_filename(body))
+      format(
+        ENTRIES_PATH_TEMPLATE,
+        type: type, name: str_to_filename(body), timestamp: Time.now.to_i
```

Comment:
A format like the one used in Rails migration files might provide better visibility and be a superior option:

```suggestion
        type: type, name: str_to_filename(body), timestamp: Time.now.strftime('%Y%m%d%H%M%S')
```

Reply:
Yeah definitely. I've updated it.